godal.h

```diff
@@ -154,6 +154,25 @@ extern "C" {
 	void godalGridCreate(cctx *ctx, char *pszAlgorithm, GDALGridAlgorithm eAlgorithm, GUInt32 nPoints, const double *padfX, const double *padfY, const double *padfZ, double dfXMin, double dfXMax, double dfYMin, double dfYMax, GUInt32 nXSize, GUInt32 nYSize, GDALDataType eType, void *pData);
 	GDALDatasetH godalGrid(cctx *ctx, const char *pszDest, GDALDatasetH hSrcDS, char **switches);
 	GDALDatasetH godalNearblack(cctx *ctx, const char *pszDest, GDALDatasetH hDstDS, GDALDatasetH hSrcDS, char **switches);
+
```

Comment:
nitpick: indentation

Reply:
We're not sure what you mean by "indentation" here, these signatures seem to be in line with the other function signatures. Did you meant that `godalGridCreate()` is too long and needs to be split / indented across multiple lines like `godalDatasetRasterIO()`?

Reply:
sorry, my bad. In the diff view I thought your additions were misaligned.